packages/core/src/util/field-info.js

```diff
@@ -9,7 +9,25 @@ export const Distinct = 'distinct';
 export const Stats = { Count, Nulls, Max, Min, Distinct };
 
 /**
- * @type {Record<string, (column: string) => AggregateNode>}
+ * @typedef {Count | Distinct | Max | Min | Nulls} Stat
+ * 
+ * @typedef {{
+ *   table: string | import('@uwdata/mosaic-sql').TableRefNode,
+ *   column: (string | import('@uwdata/mosaic-sql').ColumnRefNode) & { aggregate?: boolean },
+ *   stats?: Stat[] | Set<Stat>
+ * }} FieldInfoRequest
+ * 
+ * @typedef {{
+ *   table: FieldInfoRequest["table"],
+ *   column: string,
```

Comment:
While `FieldInfoRequest.column` is `string | ColumnRefNode`, both `getFieldInfo` and `getTableInfo` return `string` type for `column` field.